src/liquid/components/ld-button/ld-button.tsx

```diff
@@ -218,12 +218,17 @@ export class LdButton implements InnerFocusable, ClonesAttributes {
       <Tag
         {...this.clonedAttributes}
         aria-busy={hasProgress ? 'true' : undefined}
-        aria-disabled={this.disabled ? 'true' : undefined}
+        aria-disabled={
+          this.disabled || this.el.getAttribute('aria-disabled') === 'true'
```

Comment:
`this.el.ariaDisabled === 'true'` is actually always false, but I get your idea 😉 Changed.

Reply:
Hmm that's weird, the type of "this.el.ariaDisabled" is a string, not a boolean, according to TypeScript... Are you sure? :)

Reply:
<img width="386" alt="Screenshot 2022-01-04 at 12 39 20" src="https://user-images.githubusercontent.com/527049/148053724-f97fae65-730c-471f-bc26-2e8cf1d6b207.png">

I guess it can handle both. But I'll change it to type string if this is what Typescript suggests.